Select Stocks by volume Increase/script.py

```diff
@@ -0,0 +1,31 @@
+import yfinance as yf
+import pandas as pd
+
+data = input("Enter file path: ")
+df = pd.read_csv(data)
+
+stocks = input("Enter column heading of stocks: ")
+days = input("Enter number of days for comparision: ")
+timesincrease = input("Enter number of times increase in volume required: ")
+print(df[stocks])
+increased_stocks = []
+
+for stock in df[stocks]:
+    stock = stock.upper()
+    if '^' in stock:
+        pass
+    else:
+        try:
+            stock_info = yf.Ticker(stock)
+            
+            hist = stock_info.history(period=days)
+            previous_averaged_volume = hist['Volume'].iloc[1:period-1:1].mean()
```

Comment:
Remove the unused variable `period` to pass the tests

Reply:
Period here is the number of days for comparison.